packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/JavaScriptChannel.java

```diff
@@ -33,6 +36,28 @@
     this.methodChannel = methodChannel;
     this.javaScriptChannelName = javaScriptChannelName;
     this.platformThreadHandler = platformThreadHandler;
+    this.instanceId = null;
+    this.flutterApi = null;
+  }
+
+  /**
```

Comment:
This should have a [summary fragment](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)